idd/Energy+.idd.in

```diff
@@ -45619,7 +45615,131 @@ Coil:Heating:DX:VariableSpeed,
         \note quadratic curve = a + b*ffa + c*ffa**2
         \note cubic curve = a + b*ffa + c*ffa**2 + d*ffa**3
         \note ffa = Fraction of the full load Air Flow
+						
+COILSYSTEM:INTEGRATEDHEATPUMP:AIRSOURCE,
+	   \memo This object is used for air-source integrated heat pump, a collection of its working modes.
+  A1, \field Name
+       \required-field
+       \type alpha
+       \reference IntegratedHeatPumps
+       \note Unique name for this instance of an air-source integrated heat pump.
+  A2, \field Indoor Air Cooling Coil Inlet Node Name
```

Comment:
Do you need Cooling Coil in this field name?  On this one you have:

``` Indoor Air Cooling Coil Inlet Node Name```

But then below you have:

```Outdoor Air Inlet Node```

I would unify these to the minimum length that still fully defines the input.  If you can get away with just `Indoor Air Inlet Node Name` that's great.  And on that note, the below node inputs should all have a trailing "Name", right @mjwitte?  Well, whatever, they should be unified to the rest of the IDD.